assets/css/jquery-ui.css

```diff
@@ -312,7 +312,7 @@
 .ui-accordion .ui-accordion-content-active { display: block; }/* Autocomplete
 ----------------------------------*/
 .ui-autocomplete { position: absolute; cursor: default; }	
-.ui-autocomplete-loading { background: white url('images/ui-anim_basic_16x16.gif') right center no-repeat; }
+.ui-autocomplete-loading { background: white; }
```

Comment:
We can probably remove more unused stuff from these stylesheets in the future. I only removed the images now because it breaks the build, and since they have been missing already it cannot cause any issue.

Reply:
Sure! We can do it in the future. My concern was more about if we used it in someplace and would be good to fix the images. But I don't think so, right?

Reply:
I haven't found any usages for them either.